_data/committers.yml

```diff
@@ -316,6 +316,10 @@
   role: Committer
   alias: jrevels
   affiliation: Beacon Biosignals
+- name: Jeffrey Vo
+  role: Committer
+  alias: jefffrey
```

Comment:
Could you use Apache ID not GitHub ID here?
jeffreyvo?

Reply:
Fixed :+1: